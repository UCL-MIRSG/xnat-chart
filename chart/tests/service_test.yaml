templates:
  - templates/service.yaml

tests:
  - it: service manifest should match snapshot
    asserts:
      - matchSnapshot:
          path: spec
