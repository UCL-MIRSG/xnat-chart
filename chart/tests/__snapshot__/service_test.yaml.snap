service manifest should match snapshot:
  1: |
    ports:
      - name: http
        port: 80
        protocol: TCP
        targetPort: 8080
    selector:
      app.kubernetes.io/instance: RELEASE-NAME
      app.kubernetes.io/name: xnat
    type: ClusterIP
  2: |
    clusterIP: None
    ports:
      - name: http
        port: 80
        protocol: TCP
        targetPort: 8080
    selector:
      app.kubernetes.io/instance: RELEASE-NAME
      app.kubernetes.io/name: xnat
    sessionAffinity: ClientIP
    type: ClusterIP
